python_modules/dagster/docs/guides/adding_event.md

```diff
@@ -0,0 +1,90 @@
+## How to add a new event for comsumption by the GraphQL API
```

Comment:
*Fantastic* writeup. We should probably find a better place to but internal-only docs like this but this is fine for now